app/models/game.rb

```diff
@@ -109,4 +109,8 @@ def export_data(round)
         "recurring_credits" => recurring_credits,
       }
   end
+
+  def next_round
```

Comment:
If we're going to override an attribute method like this, I suggest we put converting the time zone as a conditional.

Ex.

```ruby
def next_round(in_time_zone: nil)
     if in_time_zone
        super.in_time_zone(self.time_zone)
     else
        super
     end
end
```